app/views/search/_granule_list.html.erb

```diff
@@ -97,7 +97,7 @@
               </div>
             </li>
           </ul>
-          <p class="busy panel-list-load-more" data-bind="visible: granules().isLoading()">
+          <p class="busy panel-list-load-more" data-bind="visible: granules.isLoading">
```

Comment:
It does :(  I see that now.

The issue I am having is setting isLoading to false - which should trigger the spinner to disappear - is not working for some reason.  I thought it might be a syntax error within the view.

However, that means this other code in index.html.erb must be incorrect?  Or is there any reason it would use this syntax? 

```            <p class="panel-list-load-more" data-bind="visible: collections.isLoading">
              Loading collections...
            </p>```